Source/Pokens.Pokedex/Pokens.Pokedex.Business/Pokemon/IPokemonService.cs

```diff
@@ -19,5 +19,8 @@ public interface IPokemonService
         Task ChangeStarter(string pokemonId);
 
         Task ChangeImages(string pokemonId, byte[] contentImage, string imageName);
+
+        Task DeleteImageToPokemon(string pokemonId, string imageId);
```

Comment:
DeleteImages e destul. "ToPokemon" e redundant de vreme ce esti in IPokemonService si metoda primeste un pokemonId drept parametru